css/main.css

```diff
@@ -488,3 +488,128 @@ footer {
     padding: 0px 250px 80px 210px;
   }
 }
+
+/***********************/
+/* EMAIL CAPTURE FORM */
+/***********************/
+
+.section-cta {
+  padding: 4.8rem 15.4rem;
+}
+
+.container {
+  /* max-width: 100%; */
+}
+
+.cta {
+  display: grid;
+  border-radius: 9px;
+  align-items: center;
+  justify-content: center;
+
+  box-shadow: 0 1.2rem 2.4rem rgba(0, 0, 0, 0.1);
+}
+
+.cta-text-box {
+  padding: 3.6rem 4.8rem 4.8rem;
+  color: #041484;
+}
+
+.cta .heading-secondary {
+  font-size: 2.4rem;
+  margin-bottom: 1.8rem;
+  color: inherit;
+}
+
+.cta-text {
+  font-size: 1.2rem;
+  line-height: 1.8;
+  margin-bottom: 2.4rem;
+}
+
+.cta-form {
+  display: grid;
+  grid-template-columns: 60% 30%;
+  column-gap: 3.6rem;
+  row-gap: 1.2rem;
+}
+
+.cta-form label {
+  display: block;
+  font-size: 1.2rem;
+  font-weight: 500;
+  margin-bottom: 1.2rem;
+}
+
+.cta-form input {
```

Comment:
On focus can we remove the outline or change the default styling of email input?

Reply:
Hey, @DeRaowl Added the On-focus outline around the input field. Thanks for the suggestion. I forgot to add the Accessibility feature.